pallets/automation-time/src/lib.rs

```diff
@@ -1228,10 +1221,10 @@ pub mod pallet {
 		pub fn schedule_task(
 			task: &TaskOf<T>,
 			provided_id: Vec<u8>,
-			execution_times: Vec<UnixTime>,
 		) -> Result<TaskId<T>, Error<T>> {
 			let owner_id = task.owner_id.clone();
 			let task_id = Self::generate_task_id(owner_id.clone(), provided_id.clone());
+			let execution_times = task.execution_times(vec![]);
```

Comment:
I only ever see you pass in `vec![]` here. Why does `task.execution_times` need `Schedule::Recurring` to append to a vector anyway?

Reply:
Leftover code from my struggle with rust's lifetime rules. Removed now. Thanks for flagging.